### PR TITLE
Updated EncompassRestClient.TimeOut to refresh _httpClient.

### PR DESCRIPTION
**Pull request by amadard**
Attempt at closing #120  Please review and approve or provide direction on corrections.

### CODE REVIEW

**Comment by TylerBrinkley** [comment]
I think I'd prefer to introduce a `CreateClientOptions` class which you can use to set any optional properties when creating a client. Thanks for contributing. I'll submit a code review over this anyway to provide you some feedback.

File: src/EncompassRest/EncompassRestClient.cs [hunk 1]
```diff
@@ -84,7 +84,8 @@ public static EncompassRestClient CreateFromAccessToken(string apiClientId, stri
         private readonly SemaphoreSlim _semaphoreSlim = new SemaphoreSlim(1);
 
         private HttpClient _httpClient;
-        private Loans.Loans _loans;
+		private TimeSpan _TimeOut = new TimeSpan(0, 0, 100);
```
Comment (TylerBrinkley) [inline src/EncompassRest/EncompassRestClient.cs]: For private fields use camelCasing preceded with an underscore. Also, I'd recommend using `TimeSpan.FromSeconds(100)` instead.

File: src/EncompassRest/Token/AccessToken.cs [hunk 2]
```diff
@@ -14,8 +14,10 @@ namespace EncompassRest.Token
 {
     public sealed class AccessToken : ApiObject
     {
-        private HttpClient _tokenClient;
-        private readonly string _apiClientId;
+        internal HttpClient _tokenClient;
+		internal TimeSpan _TimeOut = new TimeSpan(0, 0, 100);
```
Comment (TylerBrinkley) [inline src/EncompassRest/Token/AccessToken.cs]: For private fields use camelCasing preceded with an underscore. Also, I'd recommend using `TimeSpan.FromSeconds(100)` instead.

**Comment by amadard** [comment]
I tried to get the 4 spaces, VS is automatically putting in the tabs.  Checking how I can change that setting for this project.

The route of CreateClientOptions sounds dandy, I would like it if thee options holds all of the parameters (if reasonable), so we just have one CreateAsync method.

**Comment by amadard** [comment]
I added a .editorconfig file to enforce spaces vs tab for the solution, regardless of the loan VS settings.  I think this can be used to enforce CamelCase, as well as other conventions.

https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
https://dotnetthoughts.net/editorconfig-support-in-visual-studio-2017/

**Comment by TylerBrinkley** [comment]
I've never used the .editorconfig file. That's pretty awesome. Would you mind creating a separate PR for just that as it's unlikely we'll merge the other changes?

**Comment by amadard** [comment]
Corrected code based on feedback from TylerBrinkley, closing pull request to go a different route.

**Pull request closed without merging**